### PR TITLE
Aws install

### PR DESCRIPTION
**Pull request by neovintage**
Guide on how to install Conduit on AWS.

### CODE REVIEW

File: docs/Deploy/aws_ec2.mdx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+title: "Deploying to AWS EC2"
+sidebar_label: "AWS EC2"
+sidebar_position: 2
+---
+
+# Deploying to Amazon EC2
+
+Before you get started, Conduit should not be listening on any ports that are accessible to the public Internet. In this guide will we'll set up a Conduit instance that can be reached through the public Internet via SSH. We'll start at the top with a fresh AWS Account. In the end, the topology will look like this:
+
+![AWS EC2 Topology](/images/guides/aws-ec2-deployment.png)
```
Comment (jayjayjpg) [inline docs/Deploy/aws_ec2.mdx:11]: The build logs currently error with:

```shell
01:32:05.220 | Error: Image static/images/guides/aws-ec2-deployment.png used in docs/Deploy/aws_ec2.mdx not found.
-- | --
01:32:05.221 | at async Promise.all (index 0)
01:32:05.221 | [ERROR] Client bundle compiled with errors therefore further build is impossible.
01:32:05.263 | error Command failed with exit code 1.
01:32:05.264 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
01:32:05.278 | Failed: build command exited with code: 1
01:32:06.265 | Failed: an internal error occurred
```

and updating the url should resolve the issue

Comment (jayjayjpg) [reply docs/Deploy/aws_ec2.mdx]: I added another commit to update the image url to the one you already added in the PR, but feel free to edit!

Screen Shot 2022-07-21 at 12 53 19 PM

File: docs/Deploy/aws_ec2.mdx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+```bash
+$ export SSH_KEY=~/Downloads/conduit.pem
+$ chmod 400 $SSH_KEY
+$ ssh -i $SSH_KEY ec2-user@PUBLIC_IP_V4_ADDRESS
+```
+
+We're going to need to download the latest released version of Conduit. You can go the releases page on the Conduit repo in GitHub and find the version that you need, or you can copy and paste this command on the machine to do it for you:
+
+```shell
+$ TAG=$(curl -s https://api.github.com/repos/ConduitIO/conduit/releases/latest | grep "tag_name\": \"v[0-9.]*" | grep -oE '[.0-9]+') && curl -o conduit.tgz -L "https://github.com/ConduitIO/conduit/releases/download/v${TAG}/conduit_${TAG}_Linux_x86_64.tar.gz"
```
[the remark this answers is not in the document]
Comment (neovintage) [reply docs/Deploy/aws_ec2.mdx:65]: I didn't want to use tools that didn't exist on ppl's machines.

File: docs/Deploy/aws_ec2.mdx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+$ printf "[Unit]\nDescription=Conduit daemon\n\n[Service]\nType=simple\nUser=ec2-user\nWorkingDirectory=/var/lib/conduit\nExecStart=/usr/bin/conduit\n\n[Install]\nWantedBy=multi-user.target" >> /usr/local/lib/systemd/system/conduit.service
+```
+
+An [example](https://github.com/ConduitIO/conduit/blob/main/scripts/conduit.service) of a systemd script can also be found in the Conduit repo itself in the `/scripts` directory.
+
+Now that we have the script ready to go, we can add Conduit to `systemd`:
+
+```bash
+$ sudo systemctl enable conduit.service
```
[the remark this answers is not in the document]
Comment (neovintage) [reply docs/Deploy/aws_ec2.mdx:130]: eh.. I think this is fine. But we should have `.deb` packages in releases.

[the remark this answers is not in the document]
Comment (hariso) [reply docs/Deploy/aws_ec2.mdx:130]: :+1:   https://github.com/ConduitIO/conduit/issues/539

File: docs/Deploy/overview.mdx [hunk 4]
```diff
@@ -20,12 +32,47 @@ Where can you deploy conduit?
...
 ```shell
-docker run -d -p 8080:8080 ghcr.io/conduitio/conduit
+$ docker run -d -p 8080:8080 ghcr.io/conduitio/conduit
+```
+
+**Helpful Tip:** If you're using Docker, check out [Watchtower](https://containrrr.dev/watchtower/). After we push an update, Watchtower will keep your Conduit container updated by gracefully shutting down your existing container and restarting it with the same options used when initially deployed.
+
+## Building from source
+
+Before you can build Conduit from source, you need to have Go 1.18 or higher installed. The Go website has [comprehensive instructions](https://go.dev/doc/install) on how to get the language installed on your machine. Once installed, you can follow the rest of the instructions.
```
Comment (hariso) [inline docs/Deploy/overview.mdx:47]: How about the UI-related stuff? https://github.com/ConduitIO/conduit#build-from-source

File: docs/Deploy/overview.mdx [hunk 5]
```diff
@@ -1,17 +1,29 @@
 ---
-title: "Deploying Conduit"
-sidebar_label: "Overview"
+title: "Deploying Conduit on your machine"
+sidebar_label: "Local Deployment"
 sidebar_position: 1
 ---
 
-# Deploying Conduit
+# Deploying Conduit on your Machine
```
Comment (hariso) [inline docs/Deploy/overview.mdx:7]: Do we want to mention nightly builds here?

File: docs/Deploy/aws_ec2.mdx [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+
+## Create a new VPC
+
+Start by logging into the AWS console, in your region of choice, and navigate to the VPC service. Click the `Create VPC` button at the top of the page.
+
+![Create VPC](/images/guides/aws-ec2-deployment/create-vpc.png)
+
+We'll select the `Create VPC and More` option within the page. This option will create all the default networking needed to create a VPC that can route traffic internally to it and reach out to the internet. But, the VPC will not be able to accept connections from outside the VPC to it. You'll need to add at least one `NAT Gateway` to the VPC to make that possible. If you don't, you will not be able to access the instance via SSH.
+
+![Select NAT Gateway](/images/guides/aws-ec2-deployment/select-nat-gateway.png)
```
Comment (jayjayjpg) [inline docs/Deploy/aws_ec2.mdx:21]: Those screens are really nice for readers to follow along with the setup on AWS -- great addition! :sparkles:

File: docs/Deploy/aws_ec2.mdx [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+
+Let's SSH into the instance. With the newly created Key Pair, follow these commands. Don't forget to change `PUBLIC_IP_V4_ADDRESS` to the address that you got from the EC2 instance page and change `LOCATION_OF_PEM_FILE` to place where it exists on your machine:
+
+```bash
+$ export SSH_KEY=LOCATION_OF_PEM_FILE
+$ chmod 400 $SSH_KEY
+$ ssh -i $SSH_KEY ec2-user@PUBLIC_IP_V4_ADDRESS
+```
+
+We're going to need to download the latest released version of Conduit. You can go the releases page on the Conduit repo in GitHub and find the version that you need, or you can copy and paste this command on the machine to do it for you:
```
Comment (jayjayjpg) [inline docs/Deploy/aws_ec2.mdx:62]: What do you think about adding a direct link to the release page here? For example:

```suggestion
We're going to need to download the latest released version of Conduit. You can go [the releases page on the Conduit repo in GitHub](https://github.com/ConduitIO/conduit/releases) and find the version that you need, or you can copy and paste this command on the machine to do it for you:
```

**Review by jayjayjpg — APPROVED** [review]
This reads great! I only left one more small suggestion, but otherwise this comprehensive guide is already good to go -- thank you for putting this together so swiftly! :sparkles:

File: docs/Deploy/overview.mdx (hunk 4 above)
Comment (neovintage) [reply docs/Deploy/overview.mdx:47]: I can mention it.

File: docs/Deploy/overview.mdx (hunk 5 above)
Comment (neovintage) [reply docs/Deploy/overview.mdx:7]: mentioned it in the binary downloads

**Review by hariso — APPROVED** [review]
:rocket:

**Pull request merged**